backend/app/cmd/server.go

```diff
@@ -868,44 +875,55 @@ func (s *ServerCommand) loadEmailTemplate() (string, error) {
 	return string(file), nil
 }
 
-func (s *ServerCommand) makeNotify(dataStore *service.DataStore, authenticator *auth.Service) (*notify.Service, error) {
+// aside from notify.Service and error, returns telegram bot name which will be passed to the frontend
+func (s *ServerCommand) makeNotify(dataStore *service.DataStore, authenticator *auth.Service) (*notify.Service, string, error) {
```

Comment:
this one getting too big func, maybe split ? if you can figure a nice way to have makeNotifyAdmin and makeNotifyUsers it could be easier to digest

Reply:
I didn't figure a good way to split it, however I was able to chop around 20 lines off it in a second commit I made, I hope now it's better than before.